Jenkinsfile

```diff
@@ -5,12 +5,16 @@ def withMavenWorkspace(Closure body) {
 					artifactsPublisher(disabled: true),
 					junitPublisher(disabled: true)
 			]) {
-		withCredentials([string(credentialsId: 'ge.hibernate.org-access-key',
-				variable: 'GRADLE_ENTERPRISE_ACCESS_KEY')]) {
-			withGradle { // withDevelocity, actually: https://plugins.jenkins.io/gradle/#plugin-content-capturing-build-scans-from-jenkins-pipeline
-				body()
-			}
-		}
+		def develocityMainCredentialsId = helper.configuration.file?.develocity?.credentials?.main
+		def develocityBaseUrl = helper.configuration.file?.develocity?.url
+		withEnv(["DEVELOCITY_BASE_URL=${develocityBaseUrl}"]) {
```

Comment:
This won't work here unfortunately, the "helper" thing isn't used in this repo.

I'll push a commit.